vision/google/cloud/gapic/vision/v1/image_annotator_client.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/google/googleapis/blob/master/google/cloud/vision/v1/image_annotator.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google engineers.
+#
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+"""Accesses the google.cloud.vision.v1 ImageAnnotator API."""
+
+import collections
+import json
+import os
+import pkg_resources
+import platform
+
+from google.gax import api_callable
+from google.gax import config
+from google.gax import path_template
+import google.gax
+
+from google.cloud.gapic.vision.v1 import enums
+from google.cloud.proto.vision.v1 import image_annotator_pb2
+
+
+class ImageAnnotatorClient(object):
+    """
+    Service that performs Google Cloud Vision API detection tasks over client
+    images, such as face, landmark, logo, label, and text detection. The
+    ImageAnnotator service returns detected entities from the images.
+    """
+
+    SERVICE_ADDRESS = 'vision.googleapis.com'
+    """The default address of the service."""
+
+    DEFAULT_SERVICE_PORT = 443
+    """The default port of the service."""
+
+    # The scopes needed to make gRPC calls to all of the methods defined in
+    # this service
+    _ALL_SCOPES = ('https://www.googleapis.com/auth/cloud-platform', )
+
+    def __init__(self,
+                 service_path=SERVICE_ADDRESS,
```

Comment:
@jonparrott We should discuss this on Monday. (Even though Vision is now not as urgent, Video will go out soon.) If we want this change, I would prefer make it sooner rather than later.

(I am not entirely sure I agree with you on this one, but definitely want to discuss.)